### PR TITLE
[Connect] Add README + docstrings fix

### PR DESCRIPTION
**Pull request by mludowise-stripe**
## Summary
Adds a README to the StripeConnect SDK folder. This README will be visible in the generated docs reference as well as linked to from the main repo README.

## Motivation
https://jira.corp.stripe.com/browse/MXMOBILE-2494

## Testing
README is visible here:
https://github.com/stripe/stripe-ios/tree/mludowise/connect_README/StripeConnect

### CODE REVIEW

File: StripeConnect/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# <img src="../readme-images/Connect.svg" width="40" /> StripeConnect iOS SDK
```
Comment (nschris-stripe) [inline StripeConnect/README.md:1]: Should StripeConnect be one word or two words here? I see we typically refer to it as "Stripe Connect" below.

**Review by nschris-stripe — CHANGES_REQUESTED** [review]

File: StripeConnect/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   * [Example](#example)
+* [Manual linking](#manual-linking)
+
+<!--te-->
+
+## Supported components
+
+The following Connect embedded components are supported in the iOS SDK:
+
+* [**Account onboarding**](https://docs.stripe.com/connect/supported-embedded-components/account-management?platform=ios): Show account details and allow them to be edited.
```
[the remark this answers is not in the document]
Comment (mludowise-stripe) [reply StripeConnect/README.md]: 🤦‍♀️ good catch

**Review by nschris-stripe — DISMISSED** [review]

File: StripeConnect/README.md (hunk 1 above)
Comment (mludowise-stripe) [reply StripeConnect/README.md:1]: Yeah, I think it should be two words based on what we do for FC / Identity

Comment (mludowise-stripe) [reply StripeConnect/README.md:1]:
```suggestion
# <img src="../readme-images/Connect.svg" width="40" /> Stripe Connect iOS SDK
```

**Review by nschris-stripe — APPROVED** [review]

**Pull request merged**